catalogue/webapp/test/e2e/helpers/urls.ts

```diff
@@ -0,0 +1,14 @@
+export const baseUrl = (): string => {
+  switch (process.env.NODE_ENV) {
+    case 'production':
+      return 'https://wellcomecollection.org';
+    case 'staging':
+      return 'https://www-stage.wellcomecollection.org';
+    default:
+      return `http://localhost:3000`;
+  }
+};
+
+export const collectionsUrl = `${baseUrl()}/collections`;
```

Comment:
This URL isn't available on the catalogue app.

Reply:
yes this should be deleted.